### DISCUSSION TITLE
TypeError:[plugins].flat is not a function

### DISCUSSION THREAD

**Discussion by yangg19** [Milkdown/milkdown#405] [Q&A]
Hello! When i using Vue2, according to the dependencies version and code in the document, there is noo error at runtime, but the following error occurs when actually entering the web page: 
Error in mounted hook: "TypeError:[plugins].flat is not a function"
How to solve this problem? I hope good guys who encouter this problem can help answer it. Thanks!

我在使用Vue2时，按照文档里的依赖版本和代码，运行时没有报错，但是真正进入网页时出现了如下错误：
Error in mounted hook: "TypeError:[plugins].flat is not a function"
请问该如何解决这个问题呢？

**Comment by Saul-Mirone** [comment]
What if you input `[].flat()` in the Console panel of your browsers dev tools?

**Reply by yangg19** [reply]
i'm so happy your reply,i has tried your introduction, but the console return :Uncaught TypeError: [].flat is not a function at <anonymous>:1:4
im a noob web developer,i dont know if i do some wrong operation according by your introduction.

**Reply by Saul-Mirone** [reply]
Seems your browser havn't support [this method](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/flat), what's the browser and version are you using?

**Reply by yangg19** [reply]
im using chrome,version:67.0.3396.87

**Reply by Saul-Mirone** [reply]
I think update to a newer chrome can be helpful.

**Reply by yangg19** [reply]
thanks i will try!

**Reply by yangg19** [reply]
I'm ashamed to bother you again, but I really like your editor. I've tried for two days and still haven't found the problem. I have to ask you again.
I have updated chrome to the latest version and operated in full accordance with the instructions. The console reports an error: `uncaught (in promise) type error: cannot read properties of undefined (reading 'injectglobal')`
I located the problematic code in _/design system / lib / index es.js._ `Const CSS = emotion.injectGlobal` in the injectvar method
I hope you can see if you know where my operation is wrong when you are free，thanks!

**Reply by yangg19** [reply]
If this will take you too much time, please ignore my request and don't want my case to trouble you
